test_team_frost.py

```diff
@@ -1,5 +1,5 @@
 import unittest
-from pii_team_frost import *
+from pii_team_frost import find_instagram_handle
```

Comment:
You either need to change this back to import * or specifically import each function by name.  I would suggest reverting to import *